pkg/types/ovirt/platform.go

```diff
@@ -10,6 +10,8 @@ type Platform struct {
 	// The target network of all the network interfaces of the nodes. Omitting defaults to ovirtmgmt
 	// network which is a default network for evert ovirt cluster.
 	NetworkName string `json:"ovirt_network_name,omitempty"`
+	//VNICProfileID defines the VNIC profile ID to use the the VM network interfaces.
+	VNICProfileID string `json:"ovirt_vnic_profile_id,omitempty"`
```

Comment:
please include `// +optional` as this field will be optional.

also details about the defaults when not specified.

Reply:
done